webpack.config.js

```diff
@@ -26,6 +26,9 @@ const featureFlags = _.get(optional('./config/local'), 'featureFlags', {
   pendoEnabled: process.env.PENDO_ENABLED || true,
   clinicsEnabled: process.env.CLINICS_ENABLED || false,
 });
+const keycloakUrl = _.get(optional('./config/local'), 'keycloakUrl', process.env.KEYCLOAK_URL || null);
```

Comment:
Perhaps we should update `config/local.example.js` to include the `keycloakUrl, keycloakRealm, keycloakClientId` exports with default (falsey) values.

Reply:
Good call, adding these in.